backward-compatibility/data/kms.ron

```diff
@@ -355,4 +355,304 @@
             ),
         )),
     ),
+    (
+        kms_core_version_min: "0.12.2",
```

Comment:
something is weird here, why is the version 0.12.2 when we don't have any 0.12.x backward compatibility data generation file?